File: test/fixtures/sample-code/mixed.ts
/**
 * Already documented function.
 */
export function documented(): void {
  return;
}

export function undocumented(x: number): number {
  return x * 2;
}

/**
 * A configuration interface.
 */
export interface AppConfig {
  port: number;
  host: string;
}

export interface UndocumentedConfig {
  debug: boolean;
}

export const arrowFn = (a: string, b: string) => a + b;

File: test/fixtures/sample-code/Mixed.java
package com.example;

/**
 * Application configuration holder.
 */
public class AppConfig {
    private int port;
    private String host;

    /**
     * Returns the configured port.
     * @return the port number
     */
    public int getPort() {
        return port;
    }

    public String getHost() {
        return host;
    }
}

public interface UndocumentedService {
    void execute();
}

/**
 * Processes incoming requests.
 */
public class RequestProcessor {
    @Override
    public String toString() {
        return "RequestProcessor";
    }

    public void handleRequest(String payload) {
        System.out.println(payload);
    }
}
